doc/modules/mod_smart_markers.md

```diff
@@ -0,0 +1,35 @@
+## Module Description
+
+Smart markers are an experimental feature, described in detail as our [Open XMPP Extension](../open-extensions/smart-markers.md).
+
+## Options
+
+### `modules.mod_smart_markers.iqdisc`
```

Comment:
you have reset_markers inside iqdisc block of the docs.

Reply:
Damn copy paste xD